src/App.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import AddTodoForm from './AddTodoForm';
+import TodoList from './TodoList';
+
+const useSemiPersistentState = () => {
+  const [todoList, setTodoList] = React.useState(
+    JSON.parse(localStorage.getItem('savedTodoList'))
+  );
```

Comment:
There's a small bug here. What do you think will happen on the initial run of the app here? If there's no data in the localStorage.

Feel free to DM me for questions too